extensions/eclipse-che-theia-terminal/src/browser/terminal-widget/remote-terminal-widget.ts

```diff
@@ -37,6 +37,11 @@ export interface RemoteTerminalWidgetFactoryOptions extends Partial<TerminalWidg
     created: string
 }
 
+export interface TerminalProcessInfo {
```

Comment:
Can't this interface be reused from upstream's [base-terminal-protocol.ts](https://github.com/eclipse-theia/theia/blob/bb43e9ea7508687932e386a344f2afc0e915e451/packages/terminal/src/common/base-terminal-protocol.ts#L20-L23)?

Reply:
It can, thx! Fixed